views/update/remove/remove-is-entity-registered-owner.html

```diff
@@ -52,28 +52,28 @@ <h1 class="govuk-heading-xl govuk-!-margin-bottom-3">Is the overseas entity curr
               text: "No, it is not listed as the registered owner of UK property or land",
               checked: is_listed_as_property_owner == 0,
               attributes: {
-                "data-event-id": "no-is-listed-as-property-owner"
+                "data-event-id": "no-is-not-listed-as-property-owner"
               }
             }
           ]
         }) }}
         <details class="govuk-details" data-module="govuk-details">
           <summary class="govuk-details__summary">
-            <span class="govuk-details__summary-text">
+            <span class="govuk-details__summary-text" data-event-id="dropdown-info-how-to-check-if-oe-listed">
               How can I check if the overseas entity is still listed?
             </span>
           </summary>
           <div class="govuk-details__text">
             <p>You can check the land registry records online (they will charge a fee):</p>
             <ul class="govuk-list govuk-list--bullet">
               <li>
-                <a href="https://www.gov.uk/government/organisations/land-registry">HM Land Registry</a> (for England and Wales)
+                <a href="https://www.gov.uk/government/organisations/land-registry" class="govuk-link"data-event-id="hm-land-registry-link">HM Land Registry</a> (for England and Wales)
```

Comment:
needs a space between the class and data-event-id attributes (unless it just looks different in the github review page)
Same applies for the other links you've added

Reply:
not sure what happened here, have now pushed the changes up.